src/HipchatGateway.php

```diff
@@ -89,7 +89,9 @@ public function notify($to, $message)
 
         $params['color'] = $color;
 
-        return $this->send($this->buildUrlFromString("room/{$to}/message"), $params);
+        $type = Arr::get($this->config, 'type', 'message');
```

Comment:
Maybe we should move the default config handling out to the factory classes in all cases?